Path for apxs2 in debian is not /usr/sbin/apxs2 but /usr/bin/apxs2.
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-MY_APXS=/usr/sbin/apxs2
+MY_APXS=/usr/bin/apxs2
 MY_LDFLAGS=-lmemcache -L/mnt/distributions/rpmbuilds/ste-1.0/ste-php/TMP/SFR-libmemcache-1.4.0.rc2-build/product/sfr-suse-addon/lib
 MY_CFLAGS=-I/mnt/distributions/rpmbuilds/ste-1.0/ste-php/TMP/SFR-libmemcache-1.4.0.rc2-build/product/sfr-suse-addon/include
 
